.github/workflows/full-test.yml

```diff
@@ -122,11 +126,14 @@ jobs:
           done
       - name: Set execute permissions on run_tests.sh
         run: chmod +x .github/run_tests.sh
+      - name: Install Wine (Plexon2)
+        if: ${{ steps.modules-changed.outputs.PLEXON2_CHANGED == 'true' }}
+        uses: ./.github/actions/install-wine
       - name: Test core
         run: ./.github/run_tests.sh core
       - name: Test extractors
         if: ${{ steps.modules-changed.outputs.EXTRACTORS_CHANGED == 'true' || steps.modules-changed.outputs.CORE_CHANGED == 'true' }}
-        run: ./.github/run_tests.sh "extractors and not streaming_extractors"
+        run: ./.github/run_tests.sh "extractors and not streaming_extractors and not plexon2"
```

Comment:
Do we still need not here with the new condition that you added? Will it not be skipped automatically when wine is not installed and test it with all the other extractor when wine is  installed?

Reply:
nope, removed!